libs/web/components/src/ProposalAdvanced.tsx

```diff
@@ -34,18 +37,107 @@ export const ProposalAdvanced: FC<ProposalAdvancedProps> = ({
 			</div>
 			<br />
 			<If condition={open}>
-				<label htmlFor="proposalExtrinsic" className="text-lg">
-					Extrinsic Code
+				<label htmlFor="cennzExtrinsic" className="text-lg">
+					Extrinsic
+				</label>
+				<fieldset
+					id="cennzExtrinsic"
+					className="border-dark mb-4 inline-flex w-full items-center space-x-6 border-[3px] bg-white px-4 py-2"
+				>
+					<p className="mr-2 tracking-widest text-gray-600">api.tx.</p>
+					<Select
+						className="w-[12em] rounded border border-gray-200"
+						inputClassName="cursor-pointer"
+						placeholder="module"
+						defaultValue={proposalCall?.module}
```

Comment:
`Select` here doesn't show the arrow initially

<img width="722" alt="Screen Shot 2022-07-03 at 15 40 50" src="https://user-images.githubusercontent.com/97480229/177023574-521bf44f-be6e-4d46-8090-91ed1501a246.png">

Reply:
From what I can tell that's a "feature" of the `select` component.

I tried forcing it by adding a `ChevronDown` as an `endAdornment` but that leaves a space within the select that can't be clicked to open the dropdown

I wen't with always having a value to display, but this means no placeholders 😔